rejson/client.py

```diff
@@ -120,15 +120,26 @@ def jsonmget(self, path, *args):
         """
         pieces = []
         pieces.extend(args)
+        if not path:
+            path=Path.rootPath()
         pieces.append(str_path(path))
         return self.execute_command('JSON.MGET', *pieces)
 
+    def jsonmgetl(self, keys=[], path=Path.rootPath()):
```

Comment:
Separate PR - `jsonmget`'s signature should change (breaking bw compat) to `jsonmget(self, path=Path.rootPath(), keys=[], *args)`